src/overlay.tsx

```diff
@@ -61,6 +61,7 @@ const Overlay: React.FC<Props> = ({
                 }
             }}
             onClick={(e) => {
+                e.stopPropagation();
```

Comment:
Have you tried to use stopPropagation always in the onPointerDown event? (instead of only applying it when we are in android+chrome)

Reply:
I'm stopping the propagation unconditionally on click, not only for android+chrome.

Reply:
task to refactor overlay: https://jira.tid.es/browse/WEB-1671